src/cpp/session/modules/SessionProfiler.R

```diff
@@ -13,4 +13,54 @@
 #
 #
 
+.rs.addJsonRpcHandler("start_profiling", function(profilerOptions)
+{
+   tryCatch({
+      tempPath <- .Call("rs_profilesPath")
```

Comment:
You should prefer using the form `.Call(.rs.routines$rs_profilesPath)` here. As-is, the `.Call` forces a linear scan through all loaded DLLs, until a symbol with the name `rs_profilesPath` is found, which can be slower. (The objects in `.rs.routines` contain information on their associated library and so can be used directly)